pkg/startup/v7/data/common/etc/mdsd.d/mdsd.xml

```diff
@@ -7,7 +7,7 @@
         <Identity tenantNameAlias="ResourceName">
             <IdentityComponent name="Region">{{ .ContainerService.Location | XMLEscape }}</IdentityComponent>
             <IdentityComponent name="SubscriptionId">{{ .ContainerService.Properties.AzProfile.SubscriptionID | XMLEscape }}</IdentityComponent>
-            <IdentityComponent name="ResourceGroupName">{{ .ContainerService.Properties.AzProfile.ResourceGroup | XMLEscape }}</IdentityComponent>
+            <IdentityComponent name="ResourceGroupName">{{ .Derived.CustomerResourceGroup .ContainerService.ID | XMLEscape }}</IdentityComponent>
```

Comment:
v7 changes shouldn't be hitting master, can you target these in a separate PR to the `release-v7` branch?

Reply:
definitely shouldn't hit master - we need to think about whether we want to backport to v7 - let's discuss on scrum

Reply:
sorry, misunderstood the scope and how v7.1 patch is going to happen. The v71 explanation on the scrum explained it. I'll clean it up.